config/webhook/manager_webhook_patch.yaml

```diff
@@ -23,4 +23,4 @@ spec:
       - name: cert
         secret:
           defaultMode: 420
-          secretName: $(SERVICE_NAME)-cert
```

Comment:
@randomvariable - shouldn't this be subject varReference substitution ?

Reply:
Doesn't work with Tilt, because it injects another volume to support hot-reloading, and once that happens, kustomize can no longer do substitutions, because it doesn't support indexing into an array for variable substitution.

Reply:
sgtm